packages/cli/package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "@microsoft/teamsfx-cli",
-  "version": "0.1.0-rc.4",
-  "author": "Microsoft Corporation",
+  "version": "0.1.1",
```

Comment:
What's the policy of package version on dev branch? I think merge back might need version update.

Reply:
+1

Reply:
Thanks for the info and let's have a discussion offline since I don't have enough knowledge about this.